### DISCUSSION TITLE
What's the meaning of "single sign on" in main screen?

### DISCUSSION THREAD

**Discussion by laralem** [syphon-org/syphon#350] [Q&A] [answered]
Hi, I'm translating Syphon and don't see any info about this in GitHub and syphon.org
In other words: what's the difference between "sign on" and "single sign on"?

String in Weblate and screenshot: https://hosted.weblate.org/translate/syphon/syphon/en/?checksum=bbb2e201a6882b28

**Comment by 0x1a8510f2** [comment, marked as answer]
Single Sign On or SSO is a way of logging in to an app/service with another account. Like, for example, how one can log into Weblate using GitHub. As far as I know, this wouldn't really translate; when I did some translations I left that as English because it's a technical term.

**Reply by 0x1a8510f2** [reply]
(Wow, GH is turning into Reddit with the new threading and upvotes :P)

(Edit: Just noticed this is discussions, oops :P)

**Reply by laralem** [reply]
Thank you. I thought it was something specific to aTox. Generally this is, and should be, translated. I will put that info in Weblate.

More info to others in case someone drops here: https://en.wikipedia.org/wiki/Single_sign-on
